README.md

```diff
@@ -80,35 +80,17 @@ We briefly illustrate here how to run the *ksrates* Nextflow pipeline on the `te
-	
-        The dataset directory already contains a pre-filled *ksrates* pipeline configuration file for the oil palm example use case, `config_elaeis.txt`, therefore the above Nextflow command should directly launch the pipeline.
-
-    * Running on a compute cluster:
-	
-          nextflow run VIB-PSB/ksrates --config config_elaeis.txt -c custom_nextflow.config
-	
-        The `--config` parameter is the same as above.
-		
-        The `-c` parameter specifies a Nextflow configuration file. This file contains settings to configure the compute cluster to be used and the pipelines resources on it such as number of CPUs and amount of memory. It also now configures whether to use the *ksrates* Singularity or Docker container. 
-        The dataset directory already contains a template Nextflow configuration file called `custom_nextflow.config` that can be adapted to your resources. Other general template Nextflow configuration files can be found in the [doc](doc/source) directory in the repository.
-
-        If the Nextflow configuration file is simply named `nextflow.config`, the configuration file will be automatically recognized and used without having to specify it using the `-c` parameter.
-
-        Please see the [full documentation](https://ksrates.readthedocs.io/) and the [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html) for more detail on Nextflow configuration, e.g. for different HPC schedulers.
+   > **Note:** If this is the first time you launch the pipeline, Nextflow will first download *ksrates* and the *ksrates* Singularity container (in the directory specified by `NXF_SINGULARITY_CACHEDIR`, e.g. the launching directory).
+       
+       export NXF_SINGULARITY_CACHEDIR=$PWD
+       nextflow run VIB-PSB/ksrates --config ./config_elaeis.txt -with-singularity vibpsb/ksrates:latest
+	   
+   The `test` directory already contains a pre-filled *ksrates* pipeline configuration file (`config_elaeis.txt`) for the oil palm use case, provided through the `--config` parameter.
+   
+   The directory also contains a Nextflow configuration file (`nextflow.config`) to fine-tune executor resources and *ksrates* container settings. If needed, please adapt it according to the availbale resources; see the [full documentation](https://ksrates.readthedocs.io/) and the [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html) for more detail on Nextflow configuration, e.g. for different HPC schedulers. The file is automatically detected; alternatively, the user can specify a custom filename or path through the `-c` option. By default, the pipeline is run on a local executor.
```

Comment:
```suggestion
   The `test` directory also already contains a Nextflow configuration file template(`nextflow.config`) to configure the executor to be used (i.e., a local computer or a compute cluster) and its resources made available to Nextflow such as the number of CPUs. It also configures whether to use the *ksrates* Singularity or Docker container. This may need to be adapted to your available resources.
   See the [full documentation](https://ksrates.readthedocs.io/) and the [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html) for more detail on Nextflow configuration, e.g. for different HPC schedulers. We also provide additional, more general template Nextflow configuration files in the [doc](doc/source) directory in the repository.
   If the Nextflow configuration file is named `nextflow.config` the file is automatically detected. Alternatively, the user can specify a custom file by using the `-C` option (see `Nextflow documentation <https://www.nextflow.io/docs/latest/cli.html#hard-configuration-override>`__).
```

Reply:
I think this would fit better in a "Prepare the config files section" as you had before.